ts/features/fims/navigation/index.tsx

```diff
@@ -0,0 +1,72 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { Route, useNavigation, useRoute } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import * as React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import LoadingScreenContent from "../../../components/screens/LoadingScreenContent";
+import { OperationResultScreenContent } from "../../../components/screens/OperationResultScreenContent";
+import { isGestureEnabled } from "../../../utils/navigation";
+import {
+  fimsGetConsentsListAction,
+  fimsGetRedirectUrlAndOpenBrowserAction
+} from "../store/actions";
+import { fimsConsentsSelector } from "../store/selectors";
+
+export const FIMS_ROUTES = {
+  MAIN: "FIMS_MAIN",
+  CONSENTS: "FIMS_CONSENTS"
+} as const;
+
+export type FimsParamsList = {
+  [FIMS_ROUTES.MAIN]: undefined;
+  [FIMS_ROUTES.CONSENTS]: { ctaUrl: string };
```

Comment:
To be honest, since this is -- and will probably always be -- a navigation-wise slim flow, I prefer having it all in a single file.

While I do see the dissonance with one of my comment responses, I like narrowing it down to line count, as in "as long as the line count is low enough to not lose yourself, it's better to keep it in the same file"

What do you think? it's an irrelevant change after all, so if you still believe it's better to divide it, no probs 👍

Reply:
Ok it's fine that they stay in the same file